Application/EdFi.Ods.Common/Attributes/DescriptorExistsAttribute.cs

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.ComponentModel.DataAnnotations;
+using EdFi.Ods.Common.Dependencies;
+using EdFi.Ods.Common.Extensions;
+using EdFi.Ods.Common.Validation;
+
+namespace EdFi.Ods.Common.Attributes;
+
+public sealed class DescriptorExistsAttribute : ValidationAttribute
+{
+    private readonly string _descriptorName;
+
+    public DescriptorExistsAttribute(string descriptorName)
+    {
+        _descriptorName = descriptorName;
+    }
+
+    protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+    {
+        if (value == null)
+        {
+            return ValidationResult.Success;
```

Comment:
No, because this isn't a "Required" type of check... it's checking whether the descriptor _value_ provided actually exists in the ODS (using the local cache). If there is no value provided, there's nothing to check -- we return Success. To reduce confusion, I could perhaps rename the attribute to `DescriptorValueExists` (this would require code gen changes).